samza-core/src/main/java/org/apache/samza/task/AsyncRunLoop.java

```diff
@@ -595,10 +604,14 @@ public void onFailure(TaskCallback callback, Throwable t) {
     private final Set<SystemStreamPartition> processingSspSet;
     private final TaskName taskName;
     private final TaskInstanceMetrics taskMetrics;
+    private final Counter taskMetricsCompletedCounter;
```

Comment:
Same here. Please follow the current pattern of instantiating metric in TaskInstanceMetrics

Reply:
Done!